attributes/default.rb

```diff
@@ -40,3 +40,4 @@
 include_attribute 'visualstudio::vs2012'
 include_attribute 'visualstudio::vs2013'
 include_attribute 'visualstudio::vs2015'
+include_attribute 'visualstudio::vs2017'
```

Comment:
This is redundant, the `chef-client` run merges all files in the attributes folder anyway.